tracer/src/Datadog.Trace/Debugger/Sink/AgentBatchUploadApi.cs

```diff
@@ -20,33 +19,26 @@ internal class AgentBatchUploadApi : IBatchUploadApi
 
         private readonly IApiRequestFactory _apiRequestFactory;
         private readonly IDiscoveryService _discoveryService;
-        private readonly string _targetPath;
-        private readonly string _environment;
-        private readonly string _version;
-        private Uri _uri;
+        private readonly Uri? _snapshotUri;
 
-        private AgentBatchUploadApi(IApiRequestFactory apiRequestFactory, IDiscoveryService discoveryService, string targetPath, string environment, string version)
+        private AgentBatchUploadApi(IApiRequestFactory apiRequestFactory, IDiscoveryService discoveryService, Uri? snapshotUri)
         {
-            _version = version;
-            _environment = environment;
             _apiRequestFactory = apiRequestFactory;
             _discoveryService = discoveryService;
-            _targetPath = targetPath;
+            _snapshotUri = snapshotUri;
         }
 
         public static AgentBatchUploadApi Create(ImmutableDebuggerSettings settings, IApiRequestFactory apiRequestFactory, IDiscoveryService discoveryService)
         {
-            return new AgentBatchUploadApi(apiRequestFactory, discoveryService, settings.SnapshotsPath, settings.Environment, settings.ServiceVersion);
+            return new AgentBatchUploadApi(apiRequestFactory, discoveryService, settings.SnapshotUri);
         }
 
         public async Task<bool> SendBatchAsync(ArraySegment<byte> snapshots)
         {
-            var tags = new Dictionary<string, string> { { "env", _environment }, { "version", _version }, { "agent_version", _discoveryService.AgentVersion }, { "debugger_version", TracerConstants.AssemblyVersion } };
-            _uri ??= new Uri($"{_targetPath}/{_discoveryService.DebuggerEndpoint}{ToDDTagsQueryString(tags)}");
+            var uri = _snapshotUri ?? _apiRequestFactory.GetEndpoint(_discoveryService.DebuggerEndpoint);
```

Comment:
I noticed the deletion of `env`, `version`, `agent_version`, `debugger_version`, and `dd_tags` here, but I don't see them brought back anywhere in the PR.  Did we discover they aren't actually needed?

Reply:
We do not need them anymore, since we are using only `agent` mode, and all of those are sent via `RcmClient`

Reply:
Got it, thanks for clarifying!